packs/feat-effects/effect-harsh-judgement.json

```diff
@@ -22,53 +22,55 @@
         },
         "rules": [
             {
-                "domain": "all",
+                "domain": "perception",
```

Comment:
Unfortunately we still need the toggle for Seek, as it doesn't support TokenMark predicates.